angular/src/app/assets/lang/locale-en.json

```diff
@@ -1,10 +1,10 @@
 {
-  "plugin_version": "Wayfair plugin v1.1.4-alpha",
+  "plugin_version": "Wayfair plugin v1.1.4-alpha.13",
```

Comment:
Is the plan to wait to remove the '-alpha.13' just prior to release?

Reply:
yeah each release gets its own release branch, so there'll be a `release-1.1.4` branch where the version is set to vanilla `1.1.4` everywhere, and then we'll bump the `main` branch to being `1.1.5-alpha` once `1.1.4` ships.